iceoryx_integrationtest/package.xml

```diff
@@ -21,6 +21,22 @@
   <test_depend>iceoryx_hoofs</test_depend>
   <test_depend>iceoryx_dust</test_depend>
 
+  <test_depend>example_callbacks</test_depend>
+  <test_depend>example_callbacks_in_c</test_depend>
+  <test_depend>example_complexdata</test_depend>
+  <test_depend>example_icedelivery</test_depend>
+  <test_depend>example_icedelivery_in_c</test_depend>
+  <test_depend>example_icediscovery</test_depend>
+  <test_depend>example_icediscovery_in_c</test_depend>
+  <test_depend>example_icehello</test_depend>
+  <test_depend>example_iceoptions</test_depend>
+  <test_depend>example_request_response</test_depend>
+  <test_depend>example_request_response_in_c</test_depend>
+  <test_depend>example_singleprocess</test_depend>
+  <test_depend>example_user_header</test_depend>
+  <test_depend>example_waitset</test_depend>
+  <test_depend>example_waitset_in_c</test_depend>
+
```

Comment:
What about `example_ice_access_control`?

Reply:
@FerdinandSpitzschnueffler The `example_ice_access_control` is not tested via the iceoryx_integrationtests

Reply:
@FerdinandSpitzschnueffler Right, the example_ice_access_control was missing in the list and tests, thanks for the hint!

Reply:
@mossmaurice @FerdinandSpitzschnueffler It seems that was somehow never used as integrationtest and the test code there needs more effort in testing and refactoring, let's address tis in issue #1910.